restler/engine/transport_layer/messaging.py

```diff
@@ -200,6 +201,8 @@ def _append_to_header(message, content):
         elif self.connection_settings.include_user_agent:
             # Send the RESTler user agent only if a custom user agent is not specified
             message = _append_to_header(message, f"User-Agent: restler/{Settings().version}")
+        if self.connection_settings.include_unique_sequence_id:
+            message = _append_to_header(message, f"x-restler-sequence-id: {SequenceTracker().get_sequence_id()}")
```

Comment:
@marina-p actually, I do have a comment here after taking a closer look at the logging from my test run.

Requests sent from the garbage collector will include `x-restler-sequence-id: None` in the header. Is that desirable, or should `x-restler-sequence-id` only be added when there is a non-None sequence ID?

Reply:
It should be sent only if present - thanks for catching.